encrypted-media-respec.html

```diff
@@ -1928,14 +1912,20 @@ <h5>
                   </li><!-- Invalid input. -->
                   <li>
                     <p>
-                      If <var>content type</var> is not a [=valid media MIME type=] or is
-                      unrecognized, continue to the next iteration.
+                      Let <var>mimeType</var> be the result of running <a>parse a MIME type</a>
+                      with <var>content type</var>.
                     </p>
                   </li>
                   <li>
                     <p>
-                      Let <var>container</var> be the container type specified by <var>content
-                      type</var>.
+                      If <var>mimeType</var> is <code>failure</code> or is unrecognized,
+                      continue to the next iteration.
```

Comment:
At some point you might want to adopt https://infra.spec.whatwg.org/#iteration-continue and other Infra conventions.

Reply:
Thanks, let's do this in a separate PR.